web/public/js/controllers/manage-dojo-users-controller.js

```diff
@@ -140,26 +140,24 @@ function cdManageDojoUsersCtrl($scope, $state, $q, cdDojoService, alertService,
       },
       approvalRequired: true,
       image: userUtils.defaultAvatar('champion'),
-      text: $translate.instant('Mentor/Volunteer - these volunteers power their local Dojos with their technical and organisational skills and inspire the next generation of coders, entrepreneurs and innovators!'),
+      text: $translate.instant('Youth Champion Text...'),
```

Comment:
Youth Champion - These young people are trailblazers who set an example and inspire other young people by organising and leading their local club. Young people should participate in the Dojo they Champion, meaning  that Youth Champions improve their coding skills too!